test/examples/uct_hello_world.c

```diff
@@ -156,12 +157,18 @@ void zcopy_completion_cb(uct_completion_t *self, ucs_status_t status)
 ucs_status_t do_am_zcopy(iface_info_t *if_info, uct_ep_h ep, uint8_t id,
                          const cmd_args_t *cmd_args, char *buf)
 {
+    ucs_status_t status = UCS_OK;
     uct_mem_h memh;
     uct_iov_t iov;
     zcopy_comp_t comp;
 
-    ucs_status_t status = uct_md_mem_reg(if_info->md, buf, cmd_args->test_strlen,
-                                         UCT_MD_MEM_ACCESS_RMA, &memh);
+    if ((if_info->md_attr.cap.flags & UCT_MD_FLAG_NEED_MEMH)) {
+        status = uct_md_mem_reg(if_info->md, buf, cmd_args->test_strlen,
+                                UCT_MD_MEM_ACCESS_RMA, &memh);
+    } else {
+        memh = UCT_MEM_HANDLE_NULL;
```

Comment:
what happens when mem_dereg is called?

Reply:
yes, need to handle this
for TCP it was OK, because `uct_ep_am_zcopy` returns `UCS_OK`
thank you